### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by max1mde**
Basically just added lombok and used it

### CODE REVIEW

**Comment by max1mde** [comment]
What do you think

**Comment by corentingosselin** [comment]
> What do you think

nice one ! thanks

File: src/main/java/fr/cocoraid/prodigycape/database/MysqlDatabase.java
[the remark this answers is not in the document]
Comment (max1mde) [reply src/main/java/fr/cocoraid/prodigycape/database/MysqlDatabase.java:1]: Whats there?

[the remark this answers is not in the document]
Comment (max1mde) [reply src/main/java/fr/cocoraid/prodigycape/database/MysqlDatabase.java:1]: Here I had to rename it...

[the remark this answers is not in the document]
Comment (corentingosselin) [reply src/main/java/fr/cocoraid/prodigycape/database/MysqlDatabase.java:1]: lol okay, so ready to merge ?

[the remark this answers is not in the document]
Comment (max1mde) [reply src/main/java/fr/cocoraid/prodigycape/database/MysqlDatabase.java:1]: Can I somehow edit this in this pull request?

[the remark this answers is not in the document]
Comment (max1mde) [reply src/main/java/fr/cocoraid/prodigycape/database/MysqlDatabase.java:1]: > lol okay, so ready to merge ?

Yea it should not break anything

**Pull request merged**